app/views/avo/associations/new.html.erb

```diff
@@ -41,8 +41,14 @@
                 stacked: true,
                 classes: 'w-full'
               %>
-              <%= hidden_field_tag :turbo_frame, params[:turbo_frame] %>
             <% end %>
+            <%#
+              It is important to render this hidden field to allow selective rendering of the turbo frame.
+              (e.g. in the case of "Attach and Attach Another")
+
+              Without this, the entire page will be reloaded when the form is submitted.
+            %>
```

Comment:
@Paul-Bob I added a comment here for communication purposes. LMK if it should be reworded or if this works. Thank you for your comment - I completely missed the `hidden_field_tag`!

Reply:
Awesome! Thank you @ObiWanKeoni!